tests/testthat/test-mock-repos.R

```diff
@@ -7,113 +7,50 @@ test_that("repos, some basics", {
   skip_on_cran()
   skip_if_no_token()
 
-  res <- gh("/user/repos", .token = tt())
-  expect_true(all(c("id", "name", "full_name") %in% names(res[[1]])))
-
   res <- gh(
     TMPL("/users/{username}/repos"),
-    username = "gaborcsardi",
-    .token = tt()
+    username = "gaborcsardi"
   )
   expect_true(all(c("id", "name", "full_name") %in% names(res[[1]])))
 
   res <- gh(
     TMPL("/orgs/{org}/repos"),
     org = "r-lib",
-    type = "sources",
-    .token = tt()
+    type = "sources"
   )
   expect_true("desc" %in% vapply(res, "[[", "name", FUN.VALUE = ""))
 
-  res <- gh("/repositories", .token = tt())
+  res <- gh("/repositories")
   expect_true(all(c("id", "name", "full_name") %in% names(res[[1]])))
+})
 
-  test_repo <- basename(tempfile("gh-testing-"))
-
-  res <- gh(
-    "POST /user/repos",
-    name = test_repo,
-    description = "Test repo for gh",
-    homepage = "https://github.com/r-lib/gh",
-    private = FALSE,
-    has_issues = FALSE,
-    has_wiki = FALSE,
-    .token = tt()
-  )
-  expect_equal(res$name, test_repo)
-  expect_equal(res$description, "Test repo for gh")
-  expect_equal(res$homepage, "https://github.com/r-lib/gh")
-  expect_false(res$private)
-  expect_false(res$has_issues)
-  expect_false(res$has_wiki)
-
-  ## TODO: POST /orgs/{org}/repos
+test_that("can POST, PATCH, and DELETE", {
```

Comment:
My intuition is that these tests are really for the different verbs so I switched from modifying repos to modifying gists, which is much less invasive.